src/mixins/common-mixin.ts

```diff
@@ -77,13 +76,13 @@ export class CommonMixin extends Vue {
       case EntityTypes.SO: return CorpTypeCd.SOCIETY
       case EntityTypes.UL: return CorpTypeCd.BC_ULC_COMPANY
       case EntityTypes.XCP: return CorpTypeCd.XPRO_COOP
-      case EntityTypes.XCR: return CorpTypeCd.XPRO_CORPORATION
+      case EntityTypes.XCR: return CorpTypeCd.EXTRA_PRO_A
       case EntityTypes.XL: return CorpTypeCd.XPRO_LL_PARTNR // same as XLL
       case EntityTypes.XLL: return CorpTypeCd.XPRO_LL_PARTNR
       case EntityTypes.XLP: return CorpTypeCd.XPRO_LIM_PARTNR
       case EntityTypes.XP: return CorpTypeCd.XPRO_LIM_PARTNR // same as XLP
       case EntityTypes.XSO: return CorpTypeCd.XPRO_SOCIETY
-      case EntityTypes.XUL: return CorpTypeCd.EXTRA_PRO_A
+      case EntityTypes.XUL: return CorpTypeCd.XPRO_UNLIMITED_LIABILITY_COMPANY
```

Comment:
To be precise, Namex API seems to support XUL -> A mappings, ie

![image](https://github.com/bcgov/namerequest/assets/10002889/de40cd2f-a68e-4c87-b1d4-7e71725fed79)

However, the XUL mapping entries in Lear are currently commented out, meaning they're not supported in Lear atm, ie

![image](https://github.com/bcgov/namerequest/assets/10002889/ec6d661c-d553-4f56-ba8f-d9f4252041b3)

Interestingly, the XCR mapping entries in Lear are also commented out. This makes sense, since users need to go to COLIN to use XCR name requests... But we may still revisit this when we move over the final types from COLIN to LEAR. Good luck to us.

Reply:
A problem to keep in mind for the future haha 😅